forge-gui/res/cardsfolder/e/evolving_adaptive.txt

```diff
@@ -4,10 +4,8 @@ Types:Creature Phyrexian Warrior
 PT:0/0
 K:etbCounter:OIL:1
 S:Mode$ Continuous | Affected$ Card.Self | AddPower$ X | AddToughness$ X | Description$ CARDNAME gets +1/+1 for each oil counter on it.
-T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl+powerGTY+Other,Creature.YouCtrl+toughnessGTZ+Other | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature enters the battlefield under your control, if that creature has greater power or toughness than CARDNAME, put an oil counter on CARDNAME.
-SVar:TrigPutCounter:DB$ PutCounter | ConditionDefined$ TriggeredCard | ConditionPresent$ Creature.powerGTY+Other,Creature.toughnessGTZ+Other | CounterType$ OIL
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl+Other | Condition$ Evolve | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature enters the battlefield under your control, if that creature has greater power or toughness than CARDNAME, put an oil counter on CARDNAME.
```

Comment:
while it might be similar, technically its not evolve :/

Reply:
I know, but this avoids the most extra code without affecting anything else in the engine
We've been doing this for some others too, e. g. for Revolt on _Lulu, Loyal Hollyphant_